proposed/attributes-meta.md

```diff
@@ -0,0 +1,77 @@
+# Attributes Meta Document
+
+## 1. Summary
+
+The idea is to create a registry that holds and provides user-space attributes for PHP.
+
+This Proposal is about defining the registry and the workflows to add, modify and 
+remove attributes from that registry
+
+## 2. Why Bother?
+
+Right now libraries that want to use attributes either have to define them themselves or rely upon 
+another library that has them defined. Most of the time that also means to import all the code of 
+the other library. 
+
+Userland libraries have no interest in provising an interoperable set of attributes as they solve 
+a specific problem with *their* set of attributes. 
+
+This registry shall solve that
+
+## 3. Scope
+
+### 3.1 Goals
+
+* Define and maintain a set of attributes that are relevant for more than one
+  single tool so that different tools can use and rely upon a defined set of attributes and
+  users do not need to use several similar attributes with the same meaning for different tools.
```

Comment:
```suggestion
* Define and maintain a set of attribute definitions that are relevant for a reasonably broad portion of the PHP ecosystem so that different tools and users can use and rely upon a defined set of attributes with less duplication.
```